luci-app-filebrowser/luasrc/model/cbi/filebrowser.lua

```diff
@@ -1,42 +1,54 @@
-m = Map("filebrowser", translate("文件管理器"), translate("FileBrowser是一个基于Go的在线文件管理器，助您方便的管理设备上的文件。"))
+--[[
+
+Copyright (C) 2021 guyezi <admin@guyezi.com>
+Copyright (C) 2020 [CTCGFW] Project OpenWRT

```

Comment:
```suggestion
Copyright (C) 2020 ImmortalWrt.org
```

Project-OpenWrt 名称已弃用。